fast-list.js

```diff
@@ -585,7 +599,7 @@ function recycle(items, start, end, action) {
   var recyclableItems = [];
 
   for (var i in items) {
-    if ((i < start) || (i > end)) recyclableItems.push(i);
+    if ((i < start) || (i > end)) recyclableItems.push(~~i);
```

Comment:
we don't _need_ it, but I was surprised when I logged the result and saw strings so it made me a bit uncomfortable.

We can remove it for micro-perf concerns :)